test/functions/ldap.js

```diff
@@ -97,7 +97,7 @@ describe('ldap:', () => {
 
     expect(res.statusCode).to.equal(200);
     expect(
-      Object.keys(res.result[0]).every((key) => requiredKeys.includes(key))
+      requiredKeys.every((key) => Object.keys(res.result[0]).includes(key))
```

Comment:
Cache the `Object.keys(res.result[0])` so that `Object.keys()` doesn't execute on every iteration of `.every()`.

```suggestion
      requiredKeys.every((key) => resultKeys.includes(key))
```